core/01_core.js

```diff
@@ -561,6 +562,11 @@ for (let i = 0; i < 10; i++) {
           })`,
         );
     }
+    ObjectDefineProperty(fn, "name", {
```

Comment:
**thought:** I'm a tad bit worried about the performance effect of this.

**nitpick:** Also, this function is the wrapped async op call and not the op macro generated function itself. Both of these will now have the same name which is a tad bit odd perhaps.

From a profiling standpoint I wouldn't really mind if this was anonymous since the actual op name will still show up in the binding function that this fn calls. Alternatively, this function could perhaps be named eg. `op_name_wrapped`.

Reply:
Done